### PR TITLE
feat(nextjs): test nextjs integration

### PR DESCRIPTION
**Pull request by gfellerph**
This PR contains an experimental nextjs app that integrates the styles and some components. It's meant for testing integration into a nextjs app of all of our components, in particular the components-react package.

Run `pnpm bootstrap && pnpm nextjs` to see what this is all about.

Tasks for further development (create tickets):
- [ ] Add integration tests for react components
- [ ] Deploy next.js preview to netlify or vercel

### CODE REVIEW

**Comment by gfellerph** [comment]
Depends on https://github.com/swisspost/design-system/pull/2109

**Comment by imagoiq** [comment]
@gfellerph let me know when it's ready for review

**Comment by gfellerph** [comment]
@imagoiq I think you can have a look. I think it would be good to create separate tickets for the bigger things that come up.

File: packages/nextjs-integration/src/app/page.tsx [hunk 1]
```diff
@@ -0,0 +1,279 @@
...
+            <p>No, this is just a fantasy.</p>
+          </PostCollapsible>
+          <PostCollapsible>
+            <span slot="header">Scaramouche, Scaramouche, will you do the Fandango?</span>
+            <p>
+              Thunderbolt and lightning, very, very frightening me (Galileo) Galileo, (Galileo)
+              Galileo, Galileo Figaro, magnifico
+            </p>
+          </PostCollapsible>
+        </PostAccordion>
```
Comment (imagoiq) [inline packages/nextjs-integration/src/app/page.tsx:275]: This is not working anymore with the changes in the accordion.

**Review by imagoiq — APPROVED** [review]
LGTM.

No really important, but one of the picture return a 404 ([/big-teaser-briefmarkenausgabe-4-2023.png?mw=800&vs=1&hash=74CD9092923FB3B5BB1F55621C9A25FD](https://www.post.ch/-/media/portal-opp/teaser/big-teaser-briefmarkenausgabe-4-2023.png?mw=800&vs=1&hash=74CD9092923FB3B5BB1F55621C9A25FD))

**Pull request merged**